deploy/iso/minikube-iso/package/crio-bin/crio.conf

```diff
@@ -102,7 +102,7 @@ decryption_keys_path = "/etc/crio/keys/"
 
 # Path to the conmon binary, used for monitoring the OCI runtime.
 # Will be searched for using $PATH if empty.
-conmon = "/usr/libexec/crio/conmon"
+conmon = "/usr/bin/conmon"
```

Comment:
The path was changed?

Reply:
Yes. New buildroot include conmon by default, and the new path is this